bootstrap/environment/variables.go

```diff
@@ -163,24 +170,31 @@ func (e *Variables) buildPaths(keyMap map[string]interface{}) []string {
 	return paths
 }
 
-// buildUppercasePathMap builds a map where the key is the uppercase version of the path
-// and the value is original version of the path
-func (e *Variables) buildUppercasePathMap(paths []string) map[string]string {
-	ucMap := make(map[string]string)
+func (e *Variables) buildOverrideSpecs(paths []string) []overrideSpec {
```

Comment:
why not keep using `map[string]string`?  It could remove `getPathForMatchedOverride` and make sure the overrideName (key) is unique.

In line 126, it could be

`path, found := overrideSpecMap[envVar]`

Reply:
Very good point. THX!

Reply:
Done. See https://github.com/edgexfoundry/go-mod-bootstrap/pull/216/commits/710eb7ce820196f535f72194ecdb7dcd18992be6